<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisksForm</class>
 <widget class="QWidget" name="DisksForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>System Disk </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Select disk for system installation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abortBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>540</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>540</width>
     <height>161</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/icons/resources/disk2.png</pixmap>
   </property>
  </widget>
  <widget class="QToolButton" name="rescanBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rescan Devices&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/resources/refresh.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::DelayedPopup</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="prevBtn">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>rescanBtn</tabstop>
  <tabstop>nextBtn</tabstop>
  <tabstop>prevBtn</tabstop>
  <tabstop>abortBtn</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
